examples/hybrid/get_cache_and_tendency.jl

```diff
@@ -29,6 +29,9 @@ get_cache(
 
 function implicit_tendency!(Yₜ, Y, p, t)
     p.test_dycore_consistency && CA.fill_with_nans!(p)
+    @nvtx "precomputed quantities" color = colorant"orange" begin
```

Comment:
@dennisYatunin Is this necessary ? Or is this already done in the `tendency` assembly sequence?